src/gr/accelerators/normalHealSet.h

```diff
@@ -131,7 +131,7 @@ class IndexedNormalHealSet{
 
     _ngLength = nside2npix(resolution);
 
-    _epsilon = 1.f/_egSize;
+    _epsilon = double(1)/double(_egSize);
```

Comment:
The literal for a `double` of value `1` is `1.`, or `1.0`, or anything similar that does not have the `f` suffix:
```suggestion
    _epsilon = 1./double(_egSize);
```